### DISCUSSION TITLE
can i combine github with Repl it

### DISCUSSION THREAD

**Discussion by linoman-zz** [community/community#23715] [New to GitHub]
<p>hello dear Experts, </p>
<p>can i combine github with Repl it</p>
<p>well i just have found the online-ide_  guess that i will stick with Repl and i think it would be great if i combine it with the github-Account.  BTW: i did not have any other option - but i will try out the repl system</p>
<p>i use it with a chrome extension - this lets me integrate repl with my github-account. </p>
<p>Can i fully use the github account with repl?</p>
<p>cf - the repl - extension for chrome see: <a href="https://chrome.google.com/webstore/detail/replit/kihnihckibjknmebghcjpmemaginnipl" rel="nofollow noopener">https://chrome.google.com/webstore/detail/replit/kihnihckibjknmebghcjpmemaginnipl</a></p>
<p><span class="bbcode-b">the good thing</span>: i can work with python , php , perl and other languages</p>
<p>i will come back and report all the findings. </p>
<p>meanwhile - have a great day</p>

**Comment by mpboom** [comment, marked as answer]
<p>Hi <a class="mention" href="/linoman-zz">@linoman-zz</a> ,</p>
<p>No, I don’t think it is possible. I do have a bit of experience with the platform and haven’t noticed a feature like that. The extension you linked seems to just do importing to Repl, not exporting to GitHub.</p>
<p>But if you want to be sure, you should reach out to the Repl team.</p>

**Comment by gianmassa** [comment]
<p>Hello, I don’t know if by the time the question was made it was possible.</p>
<p>But at the moment it’s indeed possible to use github with repl.it.</p>
<p>For this you open a new repl, then you press f1 to open the shell. From there you can clone a git repository, set you git user.name and user.email, commit your changes and push it to your github repository.</p>
